corpus/data_augmentation.py

```diff
@@ -236,3 +243,167 @@ def run(self):
 
         with open(str(self.out_segment_file), "w") as segments_outfile:
             segments_outfile.writelines(segment_file_names)
+
+
+class AddImpulseResponseJob(Job):
+    """
+    Add impulse responses to each recording in the corpus with a given probability to be affected.
+    This is done by convolving the audio with an impulse response to simulate other acoustic conditions.
+    Supports .wav files
+
+    This job is dependent on the audiomentations library.
+    """
+
+    def __init__(
+        self, corpus_file, new_corpus_name, ir_path, augment_prob, time_rqmt=12
+    ):
+        """
+
+        :param Path corpus_file: Bliss corpus with wav files
+        :param str new_corpus_name: name of the new corpus
+        :param str ir_path: path to the directory containing impulse responses
+        :param float augment_prob: probability of each audio to be augmented by impulse response convolution
+        :param int time_rqmt: time requirement for the Sisypus job
+        """
+        self.corpus_file = corpus_file
+        self.new_corpus_name = new_corpus_name
+        self.impulse_responses_dir = ir_path
+
+        self.out_audio_folder = self.output_path("out_audio/", directory=True)
+        self.out_corpus = self.output_path("ir_convoluted.corpus.xml.gz")
+
+        self.ir_path = ir_path
+        self.augment_prob = augment_prob
```

Comment:
Sorting of the `__init__`: first set the inputs, then the outputs.
`ir_path` was set twice. please remove either `self.ir_path` or `self.impulse_responses_dir` and refactor below.

```suggestion
        self.corpus_file = corpus_file
        self.new_corpus_name = new_corpus_name
        self.impulse_responses_dir = ir_path
        self.augment_prob = augment_prob

        self.out_audio_folder = self.output_path("out_audio/", directory=True)
        self.out_corpus = self.output_path("ir_convoluted.corpus.xml.gz")
```